src/js/services/conversation.js

```diff
@@ -192,9 +206,16 @@ export function sendLocation(props = {}) {
                 dispatch(sendChain(postSendMessage, message))
                     .then(resolve);
             }, (err) => {
+                clearTimeout(timeout);
+                if (timedOut) {
+                    return;
+                }
                 if (err.code === LOCATION_ERRORS.PERMISSION_DENIED) {
                     setTimeout(() => alert(locationServicesDeniedText), 100);
                     dispatch(removeMessage(message._clientId));
+                } else if (err.code === LOCATION_ERRORS.POSITION_UNAVAILABLE && location.protocol !== 'https') {
```

Comment:
Chrome uses `LOCATION_ERRORS.PERMISSION_DENIED` for the `https:` problem. Will change this case to only check protocol.